src/main/frontend/util.cljc

```diff
@@ -1374,6 +1374,14 @@
                    (count val))]
        (.setRangeText input "" current (inc idx)))))
 
+#?(:cljs
+   (defn fix-open-external-with-shift!
+     [^js/MouseEvent e]
+     (when (and (.-shiftKey e) win32? (electron?)
+                (= (string/lower-case (.. e -target -nodeName)) "a")
+                (string/starts-with? (.. e -target -href) "file:"))
```

Comment:
Why the logo link will start with "file:"?
Any other cases except fixing the logo link issue?

Reply:
Because logo link is a relative path `#/`.
Sure, not only logo link has this problem, it fixed all cases that a `<a>` tag with relative route path href.